src/common.cpp

```diff
@@ -1873,6 +1884,63 @@ std::string get_executable_path(const char *argv0) {
     } else {
         return std::string(buff);
     }
+#elif defined(__OpenBSD__)
```

Comment:
Note: The $PATH path is useful for *all* systems that don't have a nicer way of figuring this out.

That bit shouldn't be hidden behind `__OpenBSD__`.